src/components/Secrets/SecretsListView/SecretsListRowTooltip.scss

```diff
@@ -0,0 +1,15 @@
+.pf-v5-c-tooltip {
+  max-width: 20%;
+  --pf-v5-c-tooltip--BackgroundColor: #fff;
+  --pf-v5-c-tooltip--Color: #000;
+  --pf-v5-c-tooltip__arrow--BackgroundColor: var(--pf-v5-c-tooltip--BackgroundColor);
+  --pf-v5-c-tooltip__content--Padding: var(--pf-v5-global--spacer--sm);
+  --pf-v5-c-tooltip__content--BackgroundColor: var(--pf-v5-c-tooltip--BackgroundColor);
+  --pf-v5-c-tooltip__content--Color: var(--pf-v5-c-tooltip--Color);
+}
+
+.pf-v5-c-tooltip__content {
+  white-space: normal;
+  word-break: break-word;
+  text-align: left;
+}
```

Comment:
We dont need to change the tooltip styles. Also, What sort of error would user see in this  tooltip?

Reply:
See the comments here: https://github.com/konflux-ci/konflux-ui/pull/229#discussion_r2095436171 regarding the tooltip styles.
For this kind of error, the tooltip currently displays the raw error.msg returned from the API or data-fetching logic  regardless of its error type.

What’s your opinion? Should we display more details like error codes, or customize the error messages to be more user-friendly?

Reply:
Currently the Error tooltip looks like:
<img width="423" alt="Screenshot 2025-05-19 at 19 25 36" src="https://github.com/user-attachments/assets/ff5c2378-91b3-4c49-ab34-29d43f58ade7" />

Reply:
done, enjoy the popover directly. currently, the raw message of Error looks like:
<img width="503" alt="Screenshot 2025-05-20 at 15 38 53" src="https://github.com/user-attachments/assets/c33175c9-0d91-4d42-8c80-ced2e7a25f44" />

Reply:
@sahil143 updated.